spec/models/event_spec.rb

```diff
@@ -82,4 +82,31 @@
       end
     end
   end
+
+  describe '.duration' do
+    subject(:event_duration) { event.reload.duration }
```

Comment:
Why not build the event with the presentations you want, like we did in the last PR instead of building the Event first and then attaching a presentation to it after the fact .

Another fun fact is that if you can get you spec to work with the `build` or `build_stubbed` factory girl method you can really speed up your test suite since you aren't having to create new records in the DB